pkg/apis/core/v1beta1/helper/errors.go

```diff
@@ -201,7 +201,10 @@ func LastErrorWithTaskID(description string, taskID string, codes ...gardencorev
 func HasNonRetryableErrorCode(lastErrors ...gardencorev1beta1.LastError) bool {
 	for _, lastError := range lastErrors {
 		for _, code := range lastError.Codes {
-			if code == gardencorev1beta1.ErrorInfraUnauthorized || code == gardencorev1beta1.ErrorConfigurationProblem {
+			if code == gardencorev1beta1.ErrorInfraUnauthorized ||
```

Comment:
You can add this detail in in `/docs/usage/shoot_status.md#error-codes` - some of the error codes there have the description `It is classified as a non-retryable error code.`. You can also add this to the doc string of the field.

Reply:
done, thank you!